sdk/core/azure-core/src/test/java/com/azure/core/util/polling/SimpleSyncPollerTests.java

```diff
@@ -485,7 +485,7 @@ public void waitUntilOperationTimesOut() {
                 return expected;
             } else if (invocationCount[0] == 1) {
                 try {
-                    Thread.sleep(2000);
+                    Thread.sleep(10000);
```

Comment:
Since these are local tests, we should be able to reduce the polling interval and not require sleeping for 10 seconds.

Reply:
The issue appears to be with low polling intervals there can be contention which results in this failing. Logically, we can see the sleep will never complete before the timeout but in some cases due to scheduling this can actually pass, resulting in the test failing.